addons/mrp_repair/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,15 @@
+---Fields in module 'mrp_repair'---
+mrp_repair   / mrp.repair               / activity_ids (one2many)       : NEW relation: mail.activity
+mrp_repair   / mrp.repair               / website_message_ids (one2many): NEW relation: mail.message
+mrp_repair   / mrp.repair.fee           / to_invoice (boolean)          : DEL 
```

Comment:
How is handled now non invoiceable repairs?

Reply:
Good question :S

Reply:
@pedrobaeza What do you mean? you still have the invoice method in the repair header, and you can set it to "no invoice".

Reply:
It's not the same. In v10, you had the option to decide on each fee line if it's going to be invoiceable or not through this field. Now on v11 all fee lines are considered invoiceable, so someone migrating from v10 that has non invoiceable fees will find that in v11, that repairs will increase the total amount without doing anything, so the option can be:

- Remove from the table the non invoiceable lines (you can move them to another "backup" table - SQL allows to do directly from the creation of the table without having to define the schema).
- Put the total amount of that lines to 0 and move the real amount to another column.

Reply:
Ok, I follow you. I would go for the second option as it is what makes more sense in new version. If you want to have a line but do not want to charge for it, you have to set its amount to 0. 

Another new different behavior to note is that a part to be removed (type "Remove" in the tab parts [technically the field is called `operations`, how confusing as the next tab is called operations too but they are `fees_lines`...:sweat:]) are never added to the invoice, while v10 they were.

Reply:
Well, then it's interesting also that you fix this. I'm personally not a user of this module in any of my customers, but it's good to let this the most "pixel perfect" done.

Reply:
Done. Please, update review!